### DISCUSSION TITLE
Значение записи А вместо АААА?

### DISCUSSION THREAD

**Discussion by ammnt** [AdguardTeam/AdGuardHome#3333] [Q&A] [answered]
Привет,

возвращается значение записи А вместо АААА:
![image](https://user-images.githubusercontent.com/45385632/125193185-d6de7c00-e253-11eb-88d9-29b6ee94a68a.png)

Смог исправить только добавив вручную rewrite на запись АААА. Так и должно быть? Не совсем понятна логика🤔

Спасибо!

**Comment by ainar-g** [comment, marked as answer]
Добрый день и поздравляю с гетом!  А какая версия AGH и как именно выглядит запись в _DNS Rewrites_?

**Reply by ammnt** [reply]
@ainar-g, версия: v0.106.3. До исправления:

![image](https://user-images.githubusercontent.com/45385632/125626900-a6b1cd92-66ac-468f-aa94-af2a58805d13.png)

Может из-за брокера? 
![image](https://user-images.githubusercontent.com/45385632/125626575-58563b3d-dccd-4a12-8477-4b6d751a2553.png)

Сейчас выглядит так и работает корректно:
![image](https://user-images.githubusercontent.com/45385632/125626715-e309e782-5860-4dd5-8a3b-5eb6b306d500.png)

Если нужно, то могу удалить этот rewrite и подготовить verbose лог.

Спасибо.

**Reply by ammnt** [reply]
@ainar-g, удалил rewrite и залогировал проблему:
![2021-07-14_16-12-31](https://user-images.githubusercontent.com/45385632/125628542-7323bf96-554b-4dae-b2a5-a46bcc708184.png)

Verbose-лог тут:
[log.txt](https://github.com/AdguardTeam/AdGuardHome/files/6816282/log.txt)

Надеюсь будет полезно. Спасибо!

**Reply by ammnt** [reply]
@ainar-g, ну так что по итогу? Баг?😀

**Reply by ainar-g** [reply]
@ammnt, прошу прощения за задержку.  Вы пробовали `beta` или `edge`?  Вполне возможно, что в beta.5 это уже починено, потому что сейчас не могу воспроизвести, а там были починены пара багов про _DNS rewrites_.

**Reply by ammnt** [reply]
@ainar-g, смогу проверить на релизе и отпишу. Спасибо!

**Reply by ammnt** [reply]
@ainar-g, теперь работает нормально. Спасибо!
